content/handbook/Interim_processes/assembly_facilitation.md

```diff
@@ -16,6 +16,7 @@ The following are some initial process templates for structuring our assemblies.
  * Opening circle 
  * Crew report-backs  
  * Peer Q&A  
+ * Rest break
```

Comment:
Adding a rest break here is fine, but on a larger issue I'm wondering if this framework is too prescriptive in the middle area, particularly around activities in small groups and reformation of crews. Maybe this structure being always the same works well, or maybe it's more a case of having a "main meeting business" items that are selected to go into each meeting

Reply:
Yeah, fair point. I was just intending for it to be indicative based on our first meeting. I'd expect whoever is facilating to adapt the specifics of each meeting. I'll try and make that clearer in the next round of edits. And/or we can come up with a more general guide.

Reply:
I've updated the file to clarify that these are example processes not a prescriptive framework